.github/workflows/ci_checks.yml

```diff
@@ -63,6 +64,25 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
 
+      - name: Cache AVD
+        uses: actions/cache@v2
+        id: avd-cache
+        with:
+          path: |
+            ~/.android/avd/*
+            ~/.android/adb*
+          key: ${{ runner.os }}-avd-27
+
+      - name: create AVD and generate snapshot for caching
```

Comment:
Not important, but I think we missed making the first letter upper case for the name